client/src/components/dummy-component.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { render, screen } from '@testing-library/react';
+
+import i18nConfig from '../../i18n/config';
+import { DummyComponent } from './dummy-component';
+
+jest.mock('react-ga');
+jest.unmock('react-i18next');
+
+describe('<DummyComponent />', () => {
+  const setup = () => {
+    render(
+      <I18nextProvider i18n={i18nConfig}>
+        <DummyComponent />
+      </I18nextProvider>
+    );
+  };
+
+  it('should render a heading', () => {
+    setup();
+
+    expect(screen.getByText(/learn to code — for free/i)).toBeInTheDocument();
+  });
+
+  it('should render a forum link', () => {
+    setup();
+
+    // screen.debug();
```

Comment:
This will print out the DOM to the terminal, and here is how our DOM looks like:

<img width="447" alt="Screen Shot 2021-08-01 at 16 19 28" src="https://user-images.githubusercontent.com/25715018/127765938-7a0549dd-dea4-4282-8c2e-0ab7aa93e331.png">